packages/builders/src/messages/embed/Assertions.ts

```diff
@@ -31,6 +31,12 @@ export const urlPredicate = s.string.url({
 	allowedProtocols: ['http:', 'https:'],
 }).nullish;
 
+export const embedAuthorPredicate = s.object({
+	name: authorNamePredicate,
+	iconURL: urlPredicate,
```

Comment:
```suggestion
	iconURL: imageURLPredicate,
```
Also can be set by attachment